scripts/build.js

```diff
@@ -135,3 +139,12 @@ module.exports = (options) => {
     if (err) throw err
   })
 }
+
+function selectEmojisBySlug(keywordSet, data) {
+  // conversion from v2 to v3 as described in https://github.com/muan/emojilib#migrating-from-2x
+  for (const emoji in data) {
+    data[emoji]['keywords'] = keywordSet[emoji]
+  }
+
+  return keyBy(data, 'slug')
```

Comment:
KeyBy needed to invert an array into a hash with a key. Looking at it again, I can probably just code it using a simple loop. Maybe even use the loop above.